README.md

```diff
@@ -1,106 +1,138 @@
-- Assuming this works we're now ready to Add some applications!!
-
-### Deploy helloworld app
-Let's deploy something, for this I'm using a pre-existing image that is essentially a helloworld HTTP application (you may wish to create your own image). First we'll need to get our image into ACR, eventually these should be generated by our own pipelines. Note you will need to be logged into Azure (az login) and change the repository name to whatever you created in the ACR step above):
-- `docker pull strm/helloworld-http`
-- `docker tag strm/helloworld-http paulsregistry80.azurecr.io/helloworld-http:v1`
-- `docker push paulsregistry80.azurecr.io/helloworld-http:v1`
-- Now copy the helloworld directory in this repository to the repository we attached to Argo above
-- Modify deployment.yaml so that the image refers to your ACR location
-- Now run `argocd app create helloworld --repo git@github.com:youruser/yourrepo.git --path helloworld --dest-server https://kubernetes.default.svc --dest-namespace default` (make sure you're logged in)
-- *Please note that the application name must be lowercase*
-- The git repo link is the same as the one we attached previously, it'll be the clone link you can see in github
-- The path is the github directory where the yaml files are for this app.
-- This create command will create in manual sync mode (I think) which means no changes will be applied until you sync it directly, we can create with *--sync-policy auto* there will be an update command too, or go update it in the UI, you can also Add the app via the UI and follow the instructions
-- We can check the state of our app within the UI or with `argocd app get helloworld`
-- There should also be a new workload in our AKS cluster in Portal named *helloworld*
-- If you used the image as above hopefully it is showing as LoadBalancer with a public IP (for testing purposes) with *Services and Ingresses* in Portal, clicking on it should open the helloworld page. If it isn't a load balancer we can patch it as we did for the argo server
-- We can then do things like change the number of replicas in the deployment yaml, we should then see the app is out of sync if we do `argocd app get helloworld` or in the UI, if we are syncing manually you can then do `argocd app sync helloworld` or use the button in the UI, you would hopefully then see this change reflected in Portal.
+
+---
+Note:
+This guide assumes the use of MacOS with permissions to be able to install various apps
+---
+
+### 1) Set up a Kubernetes Cluster
+See [here](./docs/installK8s.md)
+
+From here, we assume `kubectl` is installed/authenticated.
+
+### 2) Deploy Argo to Kubernetes
+
+These steps assume the following environment variables:
+```
+export APP_NAME=example
+export ARGO_NAMESPACE="${APP_NAME}-argocd"
```

Comment:
If not configuring for the namespace "argocd" we will need to add a note, as there will be more work

As per the Argo docs:
_Warning

The installation manifests include ClusterRoleBinding resources that reference argocd namespace. If you are installing Argo CD into a different namespace then make sure to update the namespace reference._

Or we just altering this readme to go with using argocd as the namespace